butterfree/migrations/metastore_migration.py

```diff
@@ -0,0 +1,23 @@
+"""Metastore Migration entity."""
```

Comment:
This files was not supposed to be in this PR, right?

Reply:
That's due to mypy, it'll break if I don't change this file.